tests/Form/InputWeekTest.php

```
@@ -0,0 +1,601 @@
+
+    public function testRenderWithValue(): void
+    {
+        self::assertSame(
+            <<<HTML
+            <input id="inputweek-" type="week" value="2017-W01">
+            HTML,
+            InputWeek::tag()->id('inputweek-')->value('2017-W01')->render(),
+            "Failed asserting that element renders correctly with 'value' attribute.",
+        );
+    }
+}
```

PHPMD (2.15.0)

[warning] The class InputWeekTest has 50 public methods and attributes. Consider reducing the number of public items to less than 45. (undefined)

(ExcessivePublicCount)